tensorizer/serialization.py

```diff
@@ -690,6 +693,7 @@ def __init__(
         *,
         lazy_load: bool = False,
         plaid_mode: bool = False,
+        plaid_mode_buffers: int = 4,
```

Comment:
Should make this a `ValueError` to specify when `plaid_mode` is `False` to prevent the predictable mistake of someone specifying only `plaid_mode_buffers` and expecting that to imply `plaid_mode=True`
```suggestion
        plaid_mode_buffers: Optional[int] = None,
```